### PR TITLE
Feature/new chip clickable

### PR DESCRIPTION
**Pull request by WillianLomeu**

### CODE REVIEW

File: src/components/chip-clickable/chip-clickable.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+   */
+  @Prop() disabled?: boolean = false;
+  /**
+   *  Triggered after a mouse click on close icon, return id element. Only fired when close is true.
+   */
+  @Event() chipClickableClose: EventEmitter;
+  @Event() chipClickableClick: EventEmitter;
+
+  private handleClick(event) {
+    if (event.key === 'Enter' && !this.disabled) {
```
Comment (mhnakashima) [inline src/components/chip-clickable/chip-clickable.tsx:49]: you should put this 'Enter' string in a const or make an Enum Key.

File: src/components/chip-clickable/chip-clickable.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      event.preventDefault();
+      this.chipClickableClick.emit();
+    }
+  }
+  private handleCloseChip(event) {
+    if (event.key === 'Enter' && !this.disabled) {
+      event.preventDefault();
+      this.chipClickableClose.emit({ id: this.element.id });
+      this.visible = false;
```
Comment (mhnakashima) [inline src/components/chip-clickable/chip-clickable.tsx:58]: is that correct? both cases set visible to false, if is, you should move this.visible to outside of if

File: src/components/chip-clickable/chip-clickable.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    } else {
+      event.preventDefault();
+      this.chipClickableClose.emit({ id: this.element.id });
+      this.visible = false;
+    }
+  }
+  private getSizeAvatarChip() {
+    if (this.size === 'tall') {
```
Comment (mhnakashima) [inline src/components/chip-clickable/chip-clickable.tsx:66]: create an enum for these strings

**Review by mhnakashima — CHANGES_REQUESTED** [review]

**Review by LucasMurtaTakeBlip — APPROVED** [review]

File: src/components/chip-clickable/chip-clickable.tsx (hunk 2 above)
Comment (WillianLomeu) [reply src/components/chip-clickable/chip-clickable.tsx:58]: Done!

**Pull request merged**